crates/rpc/rpc/src/eth/api/transactions.rs

```diff
@@ -43,3 +43,32 @@ where
         Ok(hash)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use reth_primitives::{hex_literal::hex, Bytes};
+    use reth_provider::test_utils::NoopProvider;
+    use reth_transaction_pool::test_utils::testing_pool;
+
+    use crate::EthApi;
+    use std::sync::Arc;
+
+    #[tokio::test]
+    async fn send_raw_transaction() {
+        let noop_provider = NoopProvider::default();
+
+        let pool = testing_pool();
+
+        let eth_api = EthApi::new(Arc::new(noop_provider), pool.clone(), ());
+
+        let tx_1 = Bytes::from(hex!("f88b8212b085028fa6ae00830f424094aad593da0c8116ef7d2d594dd6a63241bccfc26c80a48318b64b000000000000000000000000641c5d790f862a58ec7abcfd644c0442e9c201b32aa0a6ef9e170bca5ffb7ac05433b13b7043de667fbb0b4a5e45d3b54fb2d6efcc63a0037ec2c05c3d60c5f5f78244ce0a3859e3a18a36c61efb061b383507d3ce19d2"));
+
+        let tx_2 = Bytes::from(hex!("f86f8201e7852e90edd00083030d4094bd064928cdd4fd67fb99917c880e6560978d7ca1880de0b6b3a76400008025a07e833413ead52b8c538001b12ab5a85bac88db0b34b61251bb0fc81573ca093fa049634f1e439e3760265888434a2f9782928362412030db1429458ddc9dcee995"));
+
+        eth_api.send_raw_transaction(tx_1).await.unwrap();
+        eth_api.send_raw_transaction(tx_2).await.unwrap();
+
+        assert!(!pool.is_empty());
+        assert_eq!(pool.len(), 2);
```

Comment:
this is great,
I'd like an additional check that fetches the transaction via hash from pool. via `Pool::get(hash)`